bundles/org.openhab.binding.evcc/src/main/resources/OH-INF/i18n/evcc.properties

```diff
@@ -104,8 +106,3 @@ channel-type.evcc.vehicleSoC.label = Vehicle SoC
 channel-type.evcc.vehicleSoC.description = Current State of Charge of EV
 channel-type.evcc.vehicleTitle.label = Vehicle title
 channel-type.evcc.vehicleTitle.description = Name of EV
-
-# channel types
-
-offline.configuration-error.no-host = No host configured
-offline.communication-error.request-failed = Request failed
```

Comment:
And here:
https://github.com/openhab/openhab-addons/blob/fe9f0583b5d8dcc88d40c9517f63f327b61261b8/bundles/org.openhab.binding.evcc/src/main/java/org/openhab/binding/evcc/internal/EvccHandler.java#L228-L229

Reply:
See above.